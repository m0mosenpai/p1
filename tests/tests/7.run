../solution/letter-boxed tests/7.board ../dict.txt < tests/7.in
