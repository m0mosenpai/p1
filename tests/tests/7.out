Same-side letter used consecutively
